src/class-config.php

```diff
@@ -1188,613 +1114,568 @@ protected function add_field_group_fields( $field_group, $type_name, $layout = f
 	}
 
 	/**
-	 * Add field groups to Taxonomies
-	 *
-	 * @return void
+	 * Returns all available GraphQL Types
 	 */
-	protected function add_acf_fields_to_term_objects() {
+	public static function get_all_graphql_types() {
+		$graphql_types = array();
 
 		/**
-		 * Get a list of taxonomies that have been registered to show in graphql
+		 * Add post types exposed to GraphQL to GraphQL types
 		 */
-		$graphql_taxonomies = \WPGraphQL::get_allowed_taxonomies();
+		$graphql_post_types = get_post_types( ['show_in_graphql' => true] );
 
-		/**
-		 * If there are no taxonomies exposed to GraphQL, bail
-		 */
-		if ( empty( $graphql_taxonomies ) || ! is_array( $graphql_taxonomies ) ) {
-			return;
-		}
-
-		/**
-		 * Loop over the taxonomies exposed to GraphQL
-		 */
-		foreach ( $graphql_taxonomies as $taxonomy ) {
+		if ( ! empty( $graphql_post_types ) && is_array( $graphql_post_types ) ) {
 
 			/**
-			 * Get the field groups associated with the taxonomy
+			 * Prepare type key prefix and label surfix
 			 */
-			$field_groups = acf_get_field_groups(
-				[
-					'taxonomy' => $taxonomy,
-				]
-			);
+			$key_prefix = 'post_type__';
```

Comment:
Is there a specific reason you prefixed these values? 

I think it makes more sense to used the Type name from the schema directly. 

```php
[
  'graphql_types_on' => [
    'Post',
    'Page',
    'Tag',
    'Category',
    'User',
    'Comment',
  ],
]
```

etc

Reply:
Users can create custom post type or custom taxonomy with the reserved names. For example, users can't use "User" custom post type if we don't use prefix.

Reply:
Types are unique in the GraphQL Schema. You cannot create a `User` Post Type without giving it a unique GraphQL name. 

For example, if I add the following code to register a post type with the `user` GraphQL type, it won't work and there will be a debug message output:

```php
add_action( 'init', function() {

	register_post_type( 'user', [
		'show_in_graphql' => true,
		'graphql_single_name' => 'user',
		'graphql_plural_name' => 'users',
	] );

} );
```

![Screen Shot 2021-03-20 at 8 05 58 AM](https://user-images.githubusercontent.com/1260765/111872112-4520cb00-8953-11eb-8fd5-475bb0594ee7.png)

Reply:
OK. I'll remove the prefixes. Thanks for your detailed explanation!

Reply:
Any progress? Can this fix be merged into the master branch yet?

Reply:
@jasonbahl I found that we need the prefixes to group the post types and taxonomy types on front-end interface. If we don't use the prefixes, it's complicated to manage.

Reply:
Should be able to group by Interface, I imagine? 🤔

I'll think on it a bit too

Reply:
Also grouping in the UI doesn't have to be defined by the values passed to the field group definition. 

For registering ACF field groups via Code, it would be cleaner for to pass the GraphQL Types directly without having to know about extra prefixes.

Reply:
I used `input[value^="post_type__"]` and `input[value^="taxonomy__"]` selector to select all the post types and taxonomies, but if we don't use prefix, it's a bit hard to select.

Reply:
@rsm0128 👋 

I pushed a [commit](https://github.com/wp-graphql/wp-graphql-acf/pull/229/commits/f51d294d001e6a4a1846a5f5c67bda362d0c853d) with some re-factoring to the checkboxes. 

This removes the need for prefixing with `post_type__` and `taxonomy__` and uses the GraphQL Type names directly. 

I adjusted the markup to be re-usable for many interfaces, and the jQuery to also be re-usable. 

Right now we have the 2 interfaces (`ContentNode` and `TermNode`), but we'll have _at least_ one more, the `ContentTemplate` Interface, so that ACF field groups can be assigned to Templates in the Schema as well.

Reply:
@rsm0128 I pushed [one more commit](https://github.com/wp-graphql/wp-graphql-acf/pull/229/commits/0d2e19126103c72f6f39d0ef1fdb178131d18198) with the `ContentTemplate` interface added. 

Now it works like so (you can see the ContentTemplate interface included now): 

![acf-location-checkboxes](https://user-images.githubusercontent.com/1260765/113090848-e73c8080-91a7-11eb-9579-92c6c8f11955.gif)